### PR TITLE
Koenkk/zigbee2mqtt#12952: `TS110E_2gang` - add `power_on_behavior` and `switch_type` parameters

### PR DESCRIPTION
**Pull request by gorbunovav**
Fixes Koenkk/zigbee2mqtt#12952

### CODE REVIEW

File: lib/extend.js [hunk 1]
```diff
@@ -12,13 +12,38 @@ const extend = {
         return {exposes, fromZigbee, toZigbee};
     },
     light_onoff_brightness: (options={}) => {
-        options = {disableEffect: false, ...options};
```
Comment (Koenkk) [inline lib/extend.js:15]: Could you revert this formatting changes?

Comment (gorbunovav) [reply lib/extend.js:15]: @Koenkk for options only or for the whole file?

Comment (gorbunovav) [reply lib/extend.js:15]: @Koenkk there are 6 items + 2 ternary operators inside the `toZigbee` array - it is starting to get a bit hard to read 😅

**Comment by gorbunovav** [comment]
@Koenkk updated

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**